src/tests/multihost/alltests/test_sss_cache.py

```diff
@@ -0,0 +1,60 @@
+""" Automation of sss_cache tests
+
+:subsystemteam: sst_idm_sssd
+:upstream: yes
+"""
+from __future__ import print_function
+import pytest
+import ldap
+from sssd.testlib.common.utils import sssdTools, LdapOperations
+from sssd.testlib.common.expect import pexpect_ssh
+from sssd.testlib.common.exceptions import SSHLoginException
+
+
+@pytest.mark.usefixtures('setup_sssd_krb', 'create_posix_usersgroups')
+@pytest.mark.sss_cache
+class TestKrbWithLogin(object):
+    @pytest.mark.tier1_2
+    def test_sss_cache_reset(self, multihost, backupsssdconf):
+        """
+            :title: fix sss_cache to also reset cached timestamp
```

Comment:
docstrings can start from under the three quotes, it does not need to be indented to the quotes.
eg: https://github.com/SSSD/sssd/blob/master/src/tests/multihost/alltests/test_krb5.py#L21

Reply:
Done